wsagent/che-core-api-project/src/main/java/org/eclipse/che/api/project/server/ProjectService.java

```diff
@@ -908,22 +913,46 @@ public ItemReference getItem(@ApiParam(value = "Path to resource. Can be project
                 .setName(name)
                 .setText(text)
                 .setMaxItems(maxItems)
-                .setSkipCount(skipCount);
+                .setSkipCount(skipCount)
+                .setIncludePositions(true);
 
         final SearchResult result = searcher.search(expr);
         final List<SearchResultEntry> searchResultEntries = result.getResults();
-        final List<ItemReference> items = new ArrayList<>(searchResultEntries.size());
+        final List<SearchResultDto> s = new ArrayList<>(searchResultEntries.size());
```

Comment:
May be makes sense something more readable as name of variable instead of s, WDYT?